bigbluebutton-html5/imports/startup/client/logger/consoleStream.ts

```
@@ -0,0 +1,105 @@
+    logArgs.push(loggerName);
+    logArgs.push(msgCss);
+    logArgs.push(msg);
+    if (src) {
+      logArgs.push(srcCss);
+      logArgs.push(src);
+    }
+    if (Object.keys(extraFields).length) {
+      logArgs.push('\n');
+      logArgs.push(extraFields);
+    }
+    if (err && err.stack) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)